runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -187,4 +188,12 @@
   Long getLatencyTrackingInterval();
 
   void setLatencyTrackingInterval(Long interval);
+
+  @Description(
+      "Flink mode for data exchange for batch pipeline. "
+          + "Reference {@link org.apache.flink.api.common.ExecutionMode}")
+  @Default.Enum("BATCH_FORCED")
```

Comment:
I'm a bit hesitant to change the default value here. This is also used by the non-portable FlinkRunner and the default is `PIPELINED`. We haven't heard from anyone having issues with the batch execution. I'd leave this at the Flink default until we have found out the exact issue.

Reply:
Also ok for me to add a link to a JIRA issue to further investigate this issue.

Reply:
Sounds good, I will update the default.
Flink Jira for reference https://issues.apache.org/jira/browse/FLINK-10672